Ical.Net.Tests/RecurrenceTests.cs

```diff
@@ -130,19 +134,18 @@ public void DailyCount1()
             iCal,
             new CalDateTime(2006, 7, 1),
             new CalDateTime(2006, 9, 1),
-            new[]
-            {
-                new CalDateTime(2006, 07, 18, 10, 00, 00, _tzid),
-                new CalDateTime(2006, 07, 20, 10, 00, 00, _tzid),
-                new CalDateTime(2006, 07, 22, 10, 00, 00, _tzid),
-                new CalDateTime(2006, 07, 24, 10, 00, 00, _tzid),
-                new CalDateTime(2006, 07, 26, 10, 00, 00, _tzid),
-                new CalDateTime(2006, 07, 28, 10, 00, 00, _tzid),
-                new CalDateTime(2006, 07, 30, 10, 00, 00, _tzid),
-                new CalDateTime(2006, 08, 01, 10, 00, 00, _tzid),
-                new CalDateTime(2006, 08, 03, 10, 00, 00, _tzid),
-                new CalDateTime(2006, 08, 05, 10, 00, 00, _tzid)
-            },
+            [
+                new(new CalDateTime(2006, 07, 18, 10, 00, 00, _tzid), Duration.FromHours(1)),
```

Comment:
Shouldn't we reduce the size of the CalDateTime lists in RecurrenceTests to the relevant entries (which is much smaller)?

Reply:
Don't know enough about the individual tests to comment on that, but sounds like quite some work. :-) Certainly not in scope of this PR though.